src/mapeo-project.js

```diff
@@ -626,9 +628,28 @@ export class MapeoProject extends TypedEmitter {
       ])
     )
 
-    // TODO: 3. Clear data from indexes
+    // 3. Clear data from indexes
     // 3.1 Reset multi-core indexer state
+    await Promise.all(
+      Object.values(this.#dataStores)
+        .filter((dataStore) => dataStore.namespace !== 'auth')
+        .map(async (dataStore) => {
+          await dataStore.close()
+          await dataStore.unlink()
+        })
+    )
+
     // 3.2 Clear indexed data
+    const isIndexedSchema = setHas(new Set(this.#indexWriter.schemas))
+    await Promise.all(
+      Object.values(this.#dataTypes)
```

Comment:
This feels like the wrong way to access what needs to be deleted. It requires an unspoken contract of "any schema you add to the indexer must be a dataType in the this.#dataType map". I'm worried this is an easy route to a bug by a future maintainer. How about iterating the schemas from the index writer, and using the map `NAMESPACE_SCHEMAS` from DataStore to figure out which are non-auth store schemas?

Reply:
Done in 0b6aec3d3a9e726d6d5422d763dd73313589f20f.